### ISSUE TITLE
Improve the auto populate

### ISSUE THREAD

**Issue by Desertsnowman** [CalderaWP/Caldera-Forms#659]
this needs to be a little more useful.
for both post type and taxonomy I want to add:
- [ ] slug as value
- [ ] parent id as a filter
- [ ] depth ( show only parents or first child etc.. )

**Comment by Shelob9** [comment]
@Desertsnowman Good idea, but lets not add new features to 1.3.6 we're way behind already. I am punting to 1.3.7 now.

**Comment by danaskallman** [comment]
This would be great, to allow other processors to add those same labels. We are using it to integrate with [CiviCRM using this plugin](https://github.com/mecachisenros/caldera-forms-civicrm/) and starting to look at having folks add content and some data fields also get populated in their contact record in CiviCRM as a way to organize contact categories.
